File: src/main/java/no/nav/syfo/controller/domain/sykepengesoknad/oppsummering/Ledetekst.java
package no.nav.syfo.controller.domain.sykepengesoknad.oppsummering;

import java.io.Serializable;
import java.util.Map;
import static java.util.Collections.emptyMap;

public class Ledetekst implements Serializable {
    public String nokkel;
    public String tekst;
    public Map<String, String> verdier = emptyMap();

    public Ledetekst withTekst(final String tekst) {
        this.tekst = tekst;
        return this;
    }

    public Ledetekst withNokkel(final String nokkel) {
        this.nokkel = nokkel;
        return this;
    }

    public Ledetekst withVerdier(final Map<String, String> verdier) {
        this.verdier = verdier;
        return this;
    }
}

File: src/main/kotlin/no/nav/syfo/controller/domain/sykepengesoknad/oppsumering/Ledetekst.kt
package no.nav.syfo.controller.domain.sykepengesoknad.oppsumering

import java.io.Serializable

class Ledetekst : Serializable {
    var nokkel: String? = null
    var tekst: String? = null
    var verdier = emptyMap<String, String>()
    fun withTekst(tekst: String?): Ledetekst {
        this.tekst = tekst
        return this
    }

    fun withNokkel(nokkel: String?): Ledetekst {
        this.nokkel = nokkel
        return this
    }

    fun withVerdier(verdier: Map<String, String>): Ledetekst {
        this.verdier = verdier
        return this
    }
}
